diff --git a/src/read_in_parameters.cpp b/src/read_in_parameters.cpp
index 77557df..bbe8f0f 100644
--- a/src/read_in_parameters.cpp
+++ b/src/read_in_parameters.cpp
@@ -1111,7 +1111,7 @@ void check_parameters(InitData &parameter_list, std::string input_file) {
         parameter_list.useEpsFO = 1;
     }
 
-    if ((parameter_list.whichEOS > 19 && parameter_list.whichEOS != 91)
+    if ((parameter_list.whichEOS > 20 && parameter_list.whichEOS != 91)
         || parameter_list.whichEOS < 0) {
         music_message << "EOS_to_use unspecified or invalid option: "
                       << parameter_list.whichEOS;
